frb_example/pure_dart/rust/c_output_path_extra/c_output.h

```diff
@@ -109,6 +109,33 @@ typedef struct wire_ExoticOptionals {
   struct wire_NewTypeInt *newtypeint;
 } wire_ExoticOptionals;
 
+typedef struct wire_list_opt_box_autoadd_i32 {
+  int32_t **ptr;
```

Comment:
Opportunity for optimization. When sizeof(T) is smaller than a pointer, it might make sense to remove the extra layer of indirection and track null indexes in another way (either a separate list/bitmask or niche optimization), but might not be straightforward as far as performance wins are concerned.

Reply:
Looks reasonable! One way may be creating a separate issue, and gather feedback about how many people needs this performance optimization (i.e. they have bottleneck at this place), and the implement it after many people want it and it does improve speed measurably.